src/css/components/_c-form.scss

```diff
@@ -101,6 +101,27 @@ $_c-input-margin-top: 1.5rem;
 }
 
 
+.c-form__error-wrapper {
+	border: 4px solid $color-red;
+	padding: 0.5rem;
+
+	.c-form__error-list {
+		padding: 0 1rem;
+	}
+
+	.c-form__error-message {
+		font-size: 0.75rem;
+
+		a {
+			@include link-states($color-black);
+		}
+	}
+}
+
+.c-form__input[aria-invalid]{
```

Comment:
I like how you've tied appearance to state here. My preferred way to do something like this.